src/js.cookie.js

```diff
@@ -60,7 +60,7 @@
 
 				key = encodeURIComponent(String(key));
 				key = key.replace(/%(23|24|26|2B|5E|60|7C)/g, decodeURIComponent);
-				key = key.replace(/[\(\)]/g, escape);
+				key = key.replace(/[()]/g, function (s) { return '%' + s.charCodeAt(0).toString(16); });
```

Comment:
Thanks for the PR.

Yes, `escape` is deprecated an is discouraged to be used to escape URIs. In this case we are not using to escape URIs, we are just using as a convenience to encode the parens and save bytes, the same goes for the uses of `decodeURIComponent` and `encodeURIComponent` throughout the code.

The web is pretty stable, functions like `escape` are historically implemented consistently and are not going to be removed by browser vendors. The amount of bytes it saves is worth the use even though it's deprecated, we have a serious challenge here that is keeping the code base in the ~800 bytes range. If there's a problem in the functionality of the script that is solved by not using escape, then we will gladly change it.